front-end-admin/.eslintrc.js

```diff
@@ -16,16 +16,15 @@ module.exports = {
     'standard'
   ],
   // required to lint *.vue files
-  plugins: [
-    'vue'
-  ],
+  plugins: ['vue'],
   // add your custom rules here
   rules: {
     // allow async-await
     'generator-star-spacing': 'off',
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     // allow semi-colons
-    'semi': [2, 'always']
+    semi: [2, 'always'],
+    'space-before-function-paren': 0
```

Comment:
Thanks for this, this bug was really annoying and I'd love if the fix was on develop